mysql-test/suite/binlog/t/binlog_rows_query_with_filter.test

```diff
@@ -38,9 +38,9 @@ let $MYSQLD_DATADIR= `select @@datadir`;
 flush logs;
 
 echo "There should be eight rows_query events in the binlog for d1 and d2 and none for test";
-exec $MYSQL_BINLOG -vvv --skip_empty_trans --skip-gtid --database=d1 $MYSQLD_DATADIR/$master_binlog | grep Rows_query | wc -l;
-exec $MYSQL_BINLOG -vvv --skip_empty_trans --skip-gtid --database=d2 $MYSQLD_DATADIR/$master_binlog | grep Rows_query | wc -l;
-exec $MYSQL_BINLOG -vvv --skip_empty_trans --skip-gtid --database=test $MYSQLD_DATADIR/$master_binlog | grep Rows_query | wc -l;
+exec $MYSQL_BINLOG -vvv --skip_empty_trans --skip-gtids --database=d1 $MYSQLD_DATADIR/$master_binlog | grep Rows_query | wc -l;
```

Comment:
Are these changes intentional?

Reply:
No, they seem to be left over from some previous testing where --skip-gtid is deprecated in 8.0. I'll remove it.